platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -485,11 +485,19 @@ data:
   #
   #
   type: h2
+
   ## @param: url
   ## @type: string
   ## Location of the database
   #
-  url: jdbc:h2:mem:anchor-platform
+#  url: jdbc:h2:mem:anchor-platform
+
+  ## The hostname of the database server.
+  server:
```

Comment:
what would the value be for databases that don't use a server, like h2 or sqlite?

Reply:
If type is h2, the server and database will be ignore.

sqlite has a database name.

Reply:
Sounds straightforward!